test/spec/ol/resolutionconstraint.test.js

```diff
@@ -12,30 +13,30 @@ describe('ol.resolutionconstraint', function() {
         [1000, 500, 250, 100]);
     });
 
-    describe('delta 0', function() {
+    describe('direction 0', function() {
       it('returns expected resolution value', function() {
-        expect(resolutionConstraint(1000, 0, 0)).to.eql(1000);
-        expect(resolutionConstraint(500, 0, 0)).to.eql(500);
-        expect(resolutionConstraint(250, 0, 0)).to.eql(250);
-        expect(resolutionConstraint(100, 0, 0)).to.eql(100);
+        expect(resolutionConstraint(1000, 0)).to.eql(1000);
+        expect(resolutionConstraint(500, 0)).to.eql(500);
+        expect(resolutionConstraint(250, 0)).to.eql(250);
+        expect(resolutionConstraint(100, 0)).to.eql(100);
       });
     });
 
-    describe('zoom in', function() {
+    describe('direction 1', function() {
       it('returns expected resolution value', function() {
-        expect(resolutionConstraint(1000, 1, 0)).to.eql(500);
-        expect(resolutionConstraint(500, 1, 0)).to.eql(250);
-        expect(resolutionConstraint(250, 1, 0)).to.eql(100);
-        expect(resolutionConstraint(100, 1, 0)).to.eql(100);
+        expect(resolutionConstraint(1000, 1)).to.eql(1000);
+        expect(resolutionConstraint(500, 1)).to.eql(500);
+        expect(resolutionConstraint(250, 1)).to.eql(250);
+        expect(resolutionConstraint(100, 1)).to.eql(100);
       });
     });
 
-    describe('zoom out', function() {
+    describe('direction -1', function() {
       it('returns expected resolution value', function() {
-        expect(resolutionConstraint(1000, -1, 0)).to.eql(1000);
-        expect(resolutionConstraint(500, -1, 0)).to.eql(1000);
-        expect(resolutionConstraint(250, -1, 0)).to.eql(500);
-        expect(resolutionConstraint(100, -1, 0)).to.eql(250);
+        expect(resolutionConstraint(1000, -1)).to.eql(1000);
```

Comment:
Why are the results different now?

Reply:
The test suite has changed a lot since the resolution constraint signature does not include a `delta` parameter any more.

Previously: `resolutionConstraint(resolution: number, delta: number, direction: number)`
Now: `resolutionConstraint(resolution: number, direction: number, ...)`

So the "delta 0", "zoom in" and "zoom out" cases do not make sense any more. Only the direction parameter is varying